locales/tok.yml

```
@@ -0,0 +1,226 @@
+jellyfin: "Jellyfin"
+subsonic: "Subsonic"
+custom_manual: "Custom (manual API)"
+server_name_optional: "nimi pana (ken)"
+server_url_placeholder: "http://localhost:8096"
+change: "o ante"
+heart_track_to_add: "o luka e nena olin lon kute kalama la kalama li kama tawa ni"
+heart_track_to_add_server: "o luka e nena olin lon kute kalama la kalama li kama tawa ni. kin la mi pana e sona ni tawa pana"
+
+# Search & Placeholders
+search_placeholder: "Search for artists, albums or tracks..."
+search_placeholder: "o alasa e jan pali, e kulupu pi kalama sama, e kalama..."
```

YAMLlint (1.38.0)

[error] duplication of key "search_placeholder" in mapping

(key-duplicates)